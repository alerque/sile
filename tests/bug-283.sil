\begin[papersize=a6]{document}
\nofolios
\font[family=Libertinus Serif,size=12.7pt,language=tr]
Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize Tanrı'nın Rab'bimize

Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize Tanrı’nın Rab’bimize

Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının Rabbimize Tanrının
\end{document}
