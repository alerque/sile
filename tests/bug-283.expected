Set paper size 	297.6377985	419.5275636
Begin page
Mx 	34.8819
My 	29.8410
Set font 	Libertinus Serif;12.7;400;;normal;;LTR
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	96.5773
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	149.7892
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	211.4847
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	264.6965
T	51 66	(Ra)
Mx 	277.9553
T	14	(-)
Mx 	14.8819
My 	45.0810
T	67	(b)
Mx 	21.1430
T	8	(')
Mx 	23.5560
T	67 74	(bi)
Mx 	33.2588
T	78 74 91 70	(mize)
Mx 	62.2321
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	114.1034
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	174.4584
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	226.3298
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	14.8819
My 	60.3210
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	65.6230
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	124.8476
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	175.5887
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	234.8134
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	14.8819
My 	75.5610
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	76.9831
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	130.6007
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	192.7020
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	246.3196
T	51 66	(Ra)
Mx 	259.5784
T	67	(b)
Mx 	265.8395
T	8	(')
Mx 	268.2525
T	67 74	(bi)
Mx 	277.9553
T	14	(-)
Mx 	14.8819
My 	90.8010
T	78 74 91 70	(mize)
Mx 	42.1700
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	92.3561
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	151.0259
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	201.2121
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	259.8818
T	53 66 79	(Tan)
Mx 	279.5287
T	14	(-)
Mx 	14.8819
My 	106.0410
T	83 241	(rı)
Mx 	23.0480
T	8	(')
Mx 	25.4610
T	79 241 79	(nın)
Mx 	45.7339
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	104.7163
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	155.2152
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	214.1976
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	264.6965
T	51 66	(Ra)
Mx 	277.9553
T	14	(-)
Mx 	14.8819
My 	121.2810
T	67	(b)
Mx 	21.1430
T	8	(')
Mx 	23.5560
T	67 74	(bi)
Mx 	33.2588
T	78 74 91 70	(mize)
Mx 	62.2321
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	114.1034
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	174.4584
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	226.3298
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	14.8819
My 	136.5210
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	66.1342
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	125.8701
T	53 66 79 83 241 8 79 241 79	(Tanrı'nın)
Mx 	177.1224
T	51 66 67 8 67 74 78 74 91 70	(Rab'bimize)
Mx 	34.8819
My 	151.7610
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	95.0121
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	146.6586
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	206.7888
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	258.4354
T	51 66	(Ra)
Mx 	271.6942
T	67	(b)
Mx 	277.9553
T	14	(-)
Mx 	14.8819
My 	167.0010
T	1756	(’)
Mx 	18.2855
T	67 74	(bi)
Mx 	27.9883
T	78 74 91 70	(mize)
Mx 	57.2886
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	110.4776
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	172.1502
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	225.3392
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	14.8819
My 	182.2410
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	68.6043
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	130.8103
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	184.5327
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	246.7387
T	53 66 79	(Tan)
Mx 	266.3856
T	83 241	(rı)
Mx 	274.5517
T	1756	(’)
Mx 	277.9553
T	14	(-)
Mx 	14.8819
My 	197.4810
T	79 241 79	(nın)
Mx 	36.4781
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	97.7745
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	150.5873
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	211.8837
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	264.6965
T	51 66	(Ra)
Mx 	277.9553
T	14	(-)
Mx 	14.8819
My 	212.7210
T	67	(b)
Mx 	21.1430
T	1756	(’)
Mx 	24.5466
T	67 74	(bi)
Mx 	34.2494
T	78 74 91 70	(mize)
Mx 	61.9844
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	113.6081
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	173.7155
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	225.3392
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	14.8819
My 	227.9610
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	68.6043
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	130.8103
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	184.5327
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	246.7387
T	53 66 79	(Tan)
Mx 	266.3856
T	83 241	(rı)
Mx 	274.5517
T	1756	(’)
Mx 	277.9553
T	14	(-)
Mx 	14.8819
My 	243.2010
T	79 241 79	(nın)
Mx 	36.4781
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	97.7745
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	150.5873
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	211.8837
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	264.6965
T	51 66	(Ra)
Mx 	277.9553
T	14	(-)
Mx 	14.8819
My 	258.4410
T	67	(b)
Mx 	21.1430
T	1756	(’)
Mx 	24.5466
T	67 74	(bi)
Mx 	34.2494
T	78 74 91 70	(mize)
Mx 	61.9844
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	113.6081
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	173.7155
T	53 66 79 83 241 1756 79 241 79	(Tanrı’nın)
Mx 	225.3392
T	51 66 67 1756 67 74 78 74 91 70	(Rab’bimize)
Mx 	34.8819
My 	273.6810
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	92.8404
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	142.0612
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	200.0197
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	249.2406
T	51 66 67	(Rab)
Mx 	268.7605
T	67 74	(bi)
Mx 	278.4633
T	14	(-)
Mx 	14.8819
My 	288.9210
T	78 74 91 70	(mize)
Mx 	43.7857
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	93.1746
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	151.3010
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	200.6899
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	258.8164
T	53 66 79	(Tan)
Mx 	278.4633
T	14	(-)
Mx 	14.8819
My 	304.1610
T	83 241	(rı)
Mx 	23.0480
T	79 241 79	(nın)
Mx 	44.4816
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	102.4659
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	151.7125
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	209.6968
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	258.9434
T	51 66 67	(Rab)
Mx 	278.4633
T	14	(-)
Mx 	14.8819
My 	319.4010
T	67 74	(bi)
Mx 	24.5847
T	78 74 91 70	(mize)
Mx 	51.8728
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	99.6459
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	156.1567
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	203.9299
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	260.4406
T	53 66 79	(Tan)
Mx 	280.0875
T	14	(-)
Mx 	14.8819
My 	334.6410
T	83 241	(rı)
Mx 	23.0480
T	79 241 79	(nın)
Mx 	44.4816
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	102.4659
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	151.7125
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	209.6968
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	258.9434
T	51 66 67	(Rab)
Mx 	278.4633
T	14	(-)
Mx 	14.8819
My 	349.8810
T	67 74	(bi)
Mx 	24.5847
T	78 74 91 70	(mize)
Mx 	51.8728
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	99.6459
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	156.1567
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	203.9299
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	260.4406
T	53 66 79	(Tan)
Mx 	280.0875
T	14	(-)
Mx 	14.8819
My 	365.1210
T	83 241	(rı)
Mx 	23.0480
T	79 241 79	(nın)
Mx 	44.0712
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	101.6450
T	53 66 79 83 241 79 241 79	(Tanrının)
Mx 	150.4812
T	51 66 67 67 74 78 74 91 70	(Rabbimize)
Mx 	208.0550
T	53 66 79 83 241 79 241 79	(Tanrının)
End page
Finish
